src/blame.rs

```diff
@@ -0,0 +1,512 @@
+use crate::repository::Repository;
+use crate::signature::Signature;
+use napi::bindgen_prelude::*;
+use napi_derive::napi;
+use std::collections::HashSet;
+use std::convert::TryFrom;
+use std::ops::Deref;
+use std::path::Path;
+
+const MAX_SCAN_LINES: u32 = 10000;
+
+#[napi(object)]
+/// Represents a hunk of a blame operation, which is a range of lines
+/// and information about who last modified them.
+pub struct BlameHunk {
```

Comment:
I think we should define fields that respect the git2 structure as much as possible.
- Modify `signature` -> `final_signature`
- Modify `commit_id` -> `final_commit_id`
- Add `orig_signature`
- Add `orig_commit_id`